lib/security.js

```diff
@@ -128,19 +135,24 @@ const mixin = (application) => {
     password, // string
     callback // function(err, user)
   ) => {
-    const passwordHash = security.hash(password);
-    security.getUser(login, (err, user) => {
-      if (user) {
-        callback(new Error('Login already registered'), user);
+    security.hash(password, (hash, error) => {
```

Comment:
In `security.hash` you are passing an `error` into callback as a first argument, and `hash` as a second argument.